_posts/2024-12-16-edge-computing-tech.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Edge computing, some ways to make it happen
+author: msasurinen
+excerpt: >
+  In the last blog I introduced the concept of edge computing and outlined its benefits. In this blog post I want to go over the some of the enabling technologies, focusing on how your trusted cloud providers could help us in this matter. 
```

Comment:
Perhaps mention previous blog post on the intro below and have this excerpt about this current post?